src/api/ehr-out/__tests__/send-core-message.test.js

```diff
@@ -4,7 +4,8 @@ import request from 'supertest';
 import { v4 } from '../../../__mocks__/uuid';
 import app from '../../../app';
 import { sendMessage } from '../../../services/mhs/mhs-outbound-client';
-import { removeTitleFromExternalAttachments } from '../../../services/mhs/mhs-attachments-wrangler';
+import {removeTitleFromExternalAttachments, wrangleAttachments} from '../../../services/mhs/mhs-attachments-wrangler';
+import {jsonEhrExtract} from "../../health-record-transfers/__tests__/data/json-formatted-ehr-example";
```

Comment:
It is best-practice to have spaces between the curly brackets and function name you are importing.

Reply:
Good spot! Seems that IntelliJ did it automatically, fixed these in the latest commit.